src/main/java/io/github/sefiraat/slimetinker/utils/ThemeUtils.java

```diff
@@ -3,9 +3,9 @@
 import io.github.sefiraat.slimetinker.items.BaseItem;
 import io.github.sefiraat.slimetinker.utils.enums.ThemeItemType;
 import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItemStack;
+import io.github.thebusybiscuit.slimefun4.libraries.commons.lang.StringUtils;
```

Comment:
The commons-lang will be removed from Slimefun soon, so I think we need to extract those methods.